### ISSUE TITLE
Error log spam when using Hoe

### ISSUE THREAD

**Issue by Barril** [abearcodes/Valheim#53]
**Mod**: Crafting With Containers

**Mod Version**: 1.0.13

**Describe the bug**
When switching to the Hoe, I get a constant spam of errors from SetupPieceInfo.  
```
[Error  : Unity Log] InvalidOperationException: Sequence contains no elements
Stack trace:
System.Linq.Enumerable.Min (System.Collections.Generic.IEnumerable`1[T] source) (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
ABearCodes.Valheim.CraftingWithContainers.UI.UIVisualPatches.Postfix (Piece piece, UnityEngine.UI.Text ___m_buildSelection) (at <a269bd66a8044802aaeb1a2de29f79fd>:0)
(wrapper dynamic-method) Hud.DMD<Hud::SetupPieceInfo>(Hud,Piece)
Hud.UpdateBuild (Player player, System.Boolean forceUpdateAllBuildStatuses) (at <f48d6a22696245bf8d820aad6afa4fdb>:0)
(wrapper dynamic-method) Hud.DMD<Hud::Update>(Hud)
```
**How To Reproduce**
1. Switch to the Hoe

**Expected behavior**
The log doesn't fire a ton of errors

**Additional details**
The only mods I have running are unrelated, but for completeness:
* ValheimBrawler
* RememberIPDamnit

**Comment by abearcodes** [comment]
Confirmed, a bit of unexpected behaviour, fixed. Will be released in next version

**Comment by abearcodes** [comment]
Fixed in 1.0.14. Please reopen if this reoccurs.

**Comment by Barril** [comment]
Confirmed fixed.  Thank you much 👍
